src/main/webapp/resources/js/listOrders.js

```diff
@@ -0,0 +1,40 @@
+function listAllOffers() {
+    var request = new XMLHttpRequest();
+    var url = "/offers";
+    request.open("GET", url, true);
+
+    request.onload = function() {
+        var offersList = JSON.parse(request.responseText);
+        if(this.readyState == 4 && this.status == 200) {
+             for(var i = 0; i < offersList.length; i++) {
+                listOffer(offersList[i])
+             }
+        }
+    }
+    request.send();
+}
+
+function listOffer(offer) {
+    var imagePath = 'https://placehold.it/150x80?text=IMAGE';
```

Comment:
Ok, I can do this. But I will need to commit and deliver this image via git, I think it is possible.